code-challenges.test.js

```diff
@@ -34,10 +74,36 @@ const votes1 = {upVotes: 13, downVotes: 2}
 const votes2 = {upVotes: 2, downVotes: 33}
 // Expected output: -31
 
+// Pseudocode:
+// taking in one parameter, an object
+// function will take in up and down votes, compare the number of each, then return the end total of votes
+
+ describe("countVotes", () => { 
+    it("takes in an object that contains up votes and down votes and returns the end tally.", () => {
+        const votes1 = {upVotes: 13, downVotes: 2}
+        const votes2 = {upVotes: 2, downVotes: 33}
+        expect(countVotes(votes1)).toEqual(11)
+        expect(countVotes(votes2)).toEqual(-31)
+    })
+})
+
+//  ReferenceError: countVotes is not defined
 
 // b) Create the function that makes the test pass.
 
 
+// Pseudocode:
+// taking in one parameter, an object
+// function will take in up and down votes, compare the number of each, then return the end total of votes
+// destructure the object to get to the values of the keys
```

Comment:
I love the idea of destructuring the object, just don't see it in your code.  Consider refactoring down the road and implementing that idea.